lib/feed.js

```diff
@@ -197,21 +197,22 @@ Feed.prototype.confirm = function confirm_feed() {
     self.log.debug('Confirmed ' + endpoint + ': ' + self.db_safe);
     self.emit('confirm', db);
 
-    if (self.since === 'now') {
-      self.log.debug('Query since "now" is the same as query since -1');
-      self.since = -1;
-    }
-
-    if (self.since === -1) {
-      self.log.debug('Query since ' + self.since + ' will start at ' + db.update_seq);
-      self.since = db.update_seq;
-    } else if (self.since < 0) {
-      if (isNaN(db.update_seq)) {
-        return self.emit('error', new Error('DB requires specific id in "since"'));
+    // `/_db_updates` does not support since=-1, we use since="now" instead.
+    if (!self.is_db_updates) {
+      if (self.since === 'now') {
```

Comment:
The `now` string has been supported for `_changes` since CouchDB 1.3.0 and for `_db_updates` since that API was added in CouchDB 1.4.0. With that in mind, I'm sure this code doesn't really need to do this `now` -> `-1` swap, but equally I don't think it is worth spending any more time on this even if those versions are ancient and unsupported.

Reply:
I thought that too. Another PR perhaps.